components/Introduction.js

```diff
@@ -1,7 +1,37 @@
 import React from "react";
+import { Row, Col, Container } from "react-bootstrap";
 
 const Introduction = () => {
-	return <div>Introduction</div>;
+	return (
+		<Container>
+			<Row className='font-lexend flex justify-start text-center items-center m-auto'>
+				<Row
+					sm={3}
+					md={3}
+					lg={3}
+					className='text-7xl font-semibold flex justify-center w-75'
```

Comment:
dont do `w-75` instead do a percentage like `w-5/6` you can check out the available fractions on the tailwind docs